docs/components/autocomplete/index.md

```diff
@@ -148,19 +182,21 @@ export default class extends Component {
   ];
 
   @action
-  handleChange(option) {
+  onChange(option) {
     this.selected = option;
   }
 
   @action
-  handleFilter(value) {
+  onFilter(value) {
     return this.options.filter((option) => option === value);
   }
 }
 ```
 
 ## Disabled State
 
+Optional.
```

Comment:
Unlike the arguments documentation above, the documentation here is for states that reference arguments—so they're very slightly different. @ynotdraw: Should these states have an "Optional" descriptor to match the arguments documentation? Or do you think of them differently? If the former, we might make a note to add "Optional" to other components' state documentation.

Reply:
I can dig having "Optional" for these like you have in this PR 🎉

Reply:
Great! I'll open a followup.